postman/threads-api.postman_collection.json

```diff
@@ -2218,6 +2519,269 @@
+						}
+					]
+				}
+			]
+		},
+		{
+			"name": "Display Threads",
+			"item": [
+				{
+					"name": "Embed a Threads Post",
+					"request": {
+						"auth": {
+							"type": "noauth"
+						},
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{api_host}}/oembed?url=https://www.threads.net/@threads/post/DCkkKl_OGb1",
+							"host": [
+								"{{api_host}}"
+							],
+							"path": [
+								"oembed"
+							],
+							"query": [
+								{
+									"key": "url",
+									"value": "https://www.threads.net/@threads/post/DCkkKl_OGb1",
+									"description": "This is the URL of the Threads post to be embedded. With standard access, you may embed posts from the @meta, @threads, @instagram, and @facebook accounts."
+								},
+								{
+									"key": "access_token",
+									"value": "TH|{{app_id}}|{{app_secret}}",
+									"description": "This is your app access token. This consists of your app's ID and secret.",
+									"disabled": true
+								},
+								{
+									"key": "access_token",
+									"value": "{{app_access_token}}",
+									"description": "This is your app access token received via the GET /oauth/access_token endpoint. See the Authorization folder for details.",
+									"disabled": true
+								}
+							]
+						}
+					},
+					"response": [
+						{
+							"name": "Embed a Threads Post",
+							"originalRequest": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{api_host}}/oembed?url=https://www.threads.net/@threads/post/DCkkKl_OGb1",
+									"host": [
+										"{{api_host}}"
+									],
+									"path": [
+										"oembed"
+									],
+									"query": [
+										{
+											"key": "url",
+											"value": "https://www.threads.net/@threads/post/DCkkKl_OGb1",
+											"description": "This is the URL of the Threads post to be embedded. With standard access, you may embed posts from the @meta, @threads, @instagram, and @facebook accounts."
+										},
+										{
+											"key": "access_token",
+											"value": "TH|{{app_id}}|{{app_secret}}",
+											"description": "This is your app access token. This consists of your app's ID and secret.",
+											"disabled": true
+										},
+										{
+											"key": "access_token",
+											"value": "{{app_access_token}}",
+											"description": "This is your app access token received via the GET /oauth/access_token endpoint. See the Authorization folder for details.",
+											"disabled": true
+										}
```

Comment:
Same as comment above. Let's remove the duplicate `access_token`.

![image](https://github.com/user-attachments/assets/9e6b2c79-f35a-42d6-973f-a1041a3ecc31)

Reply:
See above answer